lighthouse-core/test/audits/dobetterweb/link-blocking-first-paint-test.js

```diff
@@ -39,10 +39,16 @@ describe('Block First Paint audit', () => {
   });
 
   it('fails when there are links found which block first paint', () => {
+    const linkDetails = {
+      href: 'http://google.com/css/style.css',
+      disabled: false,
+      media: '',
+      rel: 'stylesheet'
+    };
     const auditResult = LinkBlockingFirstPaintAudit.audit({
       LinksBlockingFirstPaint: {
         items: [{
-          url: 'http://google.com/style.css',
+          link: linkDetails,
           transferSize: 100,
           spendTime: 100
         }],
```

Comment:
wanna add a second link in here that's disabled? that'll strengthen the the `length === 1` assertion below.